wiki/osu!_Program_Files/User_configuration_file/fr.md

```diff
@@ -0,0 +1,230 @@
+
+***[Click to return to Main page](Osu!_Program_Files "wikilink")***
```

Comment:
~~(the link text can be translated, it just tells the reader that they can go back to the main-er page)~~
the link is actually `***[Click to return to Main page](../)***`